cores/esp8266/spiffs/spiffs_config.h

```diff
@@ -326,6 +326,17 @@ typedef uint8_t u8_t;
 #define SPIFFS_IX_MAP                         1
 #endif
 
+// By default SPIFFS in some cases relies on the property of NOR flash that bits
+// cannot be set from 0 to 1 by writing and that controllers will ignore such
+// bit changes. This results in fewer reads as SPIFFS can in some cases perform
+// blind writes, with all bits set to 1 and only those it needs reset set to 0.
+// Most of the chips and controllers allow this behavior, so the default is to
+// use this technique. If your controller is one of the rare ones that don't,
+// turn this option on and SPIFFS will perform a read-modify-write instead.
+#ifndef SPIFFS_NO_BLIND_WRITES
+#define SPIFFS_NO_BLIND_WRITES                0
```

Comment:
Hmm looks like we have also our PUYA patch in a single define here, or am I missing something?

Reply:
> or am I missing something?

This applies to spiffs directly, it's meant for other platforms. We have it disabled, because our patch is implemented at a lower level: at flash read/write. If we were to just use this one, we wouldn't be able to use eeprom or ota, etc.

Reply:
Ah yes EEPROM is the one I forgot.
I am not entirely sure we also need it for OTA, unless there are blocks partially updated during the OTA. Otherwise the data to be written is also going to be written on PUYA chips.